examples/ICESat-2_DAAC_DataAccess_Example.ipynb

```diff
@@ -2,7 +2,6 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.            cycles=['03','04','05','06','07'], tracks=['0042','0050'])
</pre><p>the searched data not contains tracks 0849 and 0902, and the errors will occurs in the following code. Thus I fix the selected tracks to 0042 and 0050. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/icesat2py/icepyx/pull/223/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>